deployment/plur_push_service/templates/certificate.yaml

```
@@ -0,0 +1,28 @@
+    size: 2048
+  duration: 2160h # 90d
+  renewBefore: 360h # 15d
+  isCA: false
+  usages:
+    - server auth
+    - client auth
+  subject:
+    organizations:
+      - cert-manager
+  dnsNames:
+  - plurpush.verse.app
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)